### PR TITLE
Additional icon added for oxd-select-button

### PR DESCRIPTION
**Pull request by MifdhaM**
## Checklist

- [x] Test Coverage is 100% for the newly added code
- [x] Storybook stories are added/updated for the changed areas
- [x] Components standards defined [in this document](https://docs.google.com/document/d/16_Nd3VxE_lTD9pVkONQ0egn7IiwyX1pVXZjzl-V4tU8/) are followed
- [x] Code is linted properly
- [x] Developer testing is done for the affected areas
- [x] Package version updated (not applicable to ent branch)
- [x] Changelog.md updated on possible breaking (applicable to ent branch)

### CODE REVIEW

File: storybook/stories/core/components/Input/SelectInputButton.stories.js [hunk 1]
```diff
@@ -132,13 +132,15 @@ Default.args = {
   button: buttonObject,
   moreIconName: 'oxd-info',
   doubleLineLabel: true,
+  additionalIconName: 'oxd-best-match'
```
Comment (Super-Chama) [inline storybook/stories/core/components/Input/SelectInputButton.stories.js:135]: no need to add it to every story. just create a new story where additional icon is visible

File: components/src/core/components/Input/Select/SelectInputButton.vue [hunk 2]
```diff
@@ -50,6 +50,21 @@
               />
               <span>{{ $vt(buttonData.labelMini) }}</span>
             </div>
+            <div
+              class="d-flex
```
Comment (Super-Chama) [inline components/src/core/components/Input/Select/SelectInputButton.vue:54]: what happens if these props are not passed? probably will need to render this conditionally if `additionalIconName` prop is set

File: components/src/core/components/Input/Select/SelectInputButton.vue [hunk 3]
```diff
@@ -50,6 +50,21 @@
...
+                justify-center oxd-select-info-button-container"
+            >
+              <oxd-icon
+                v-if="additionalIconName"
+                :tooltip="$vt(additionalIconTooltip)"
+                :flow="moreTooltipFlow"
+                class="oxd-select-info-button"
+                :size="additionalIconSize"
+                :name="additionalIconName"
+                @click="$emit('onAdditionalIconClick')"
```
Comment (Super-Chama) [inline components/src/core/components/Input/Select/SelectInputButton.vue:65]: please define this event in emits array

**Review by Super-Chama — CHANGES_REQUESTED** [review]

File: components/src/core/components/Input/Select/SelectInputButton.vue [hunk 4]
```diff
@@ -257,6 +272,17 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    additionalIconName: {
```
[the remark this answers is not in the document]
Comment (MifdhaM) [reply components/src/core/components/Input/Select/SelectInputButton.vue:276]: Done

File: storybook/stories/core/components/Input/SelectInputButton.stories.js (hunk 1 above)
Comment (MifdhaM) [reply storybook/stories/core/components/Input/SelectInputButton.stories.js]: Done. A new story is created

File: components/src/core/components/Input/Select/SelectInputButton.vue (hunk 2 above)
Comment (MifdhaM) [reply components/src/core/components/Input/Select/SelectInputButton.vue:54]: rendered conditionally based on passing the additional icon

File: components/src/core/components/Input/Select/SelectInputButton.vue (hunk 3 above)
Comment (MifdhaM) [reply components/src/core/components/Input/Select/SelectInputButton.vue:65]: Done

File: storybook/stories/core/components/Input/SelectInputButton.stories.js [hunk 5]
```diff
@@ -183,3 +184,11 @@ CustomDropdownTriggerStyles.args = {
...
   dropdownTriggerButtonBackgroundColor: '#ff0000',
 };
+
+export const WithAdditionalIcon = Template.bind({});
+WithAdditionalIcon.args = {
+  options: options,
+  button: buttonObject,
+  additionalIconName: 'oxd-best-match',
+  doubleLineLabel: true,
+};
```
Comment (Super-Chama) [inline storybook/stories/core/components/Input/SelectInputButton.stories.js:194]: please add new line at end

**Review by Super-Chama — APPROVED** [review]

File: storybook/stories/core/components/Input/SelectInputButton.stories.js (hunk 5 above)
Comment (MifdhaM) [reply storybook/stories/core/components/Input/SelectInputButton.stories.js]: Done

**Pull request merged**